platforms/web/vite.config.ts

```diff
@@ -59,6 +59,7 @@ export default defineConfig({
         onConsoleLog: (log) => {
             if (log.includes('wasm')) return false;
         },
+        teardownTimeout: 10000,
```

Comment:
Single line configuration change

Reply:
Is the problem related to vitest-dev/vitest#2008?

Maybe we can add a comment here to remind us to revisit this config later once any related issue is fixed?